### ISSUE TITLE
Use of aria-owns instead of aria-controls in combobox 1.0 pattern.

### ISSUE THREAD

**Issue by cpandhi** [w3c/aria-practices#938]
The aria 1.0 examples in the ARIA 1.1 practices use aria-owns on the input textbox which has the role of combobox. But the 1.0 combobox does not support children and hence should not be using aria-owns, it should be using aria-controls.
Incorrect example:
```
<input id="cb1-input"
           class="cb_edit"
           role="combobox"
           aria-autocomplete="both"
           aria-expanded="false"
           aria-haspopup="true"
           aria-owns="lb1"
           type="text">
```
Should be corrected to use aria-controls as most accessible design pattern :
```
<input id="cb1-input"
           class="cb_edit"
           role="combobox"
           aria-autocomplete="both"
           aria-expanded="false"
           aria-haspopup="true"
           aria-controls="lb1"
           type="text">

```

**Comment by mcking65** [comment]
@cpandhi,

From the ARIA 1.1 spec:

> The ARIA 1.0 specification describes a combobox pattern where a text input element has the combobox role and owns a listbox element. User agents, assistive technologies, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional.

To make an ARIA 1.0 combobox, where the input has role combobox, aria-owns is expected.

For ARIA 1.1, Bryan and I did propose simply changing the spec to work the way you suggest, but that was rejected by the working group.

Please feel free to re-open this issue if I have not sufficiently addressed your feedback.
